e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/HotNodePoolsAO.java

```diff
@@ -0,0 +1,230 @@
+
+    public static class CreateHotNodePoolAO extends PopupAO<CreateHotNodePoolAO, HotNodePoolsAO> {
+
+        private final Map<Primitive, SelenideElement> elements = initialiseElements(
+                entry(CLOSE, context().find(byClassName("ant-modal-close"))),
+                entry(CANCEL, context().find(button("CANCEL"))),
+                entry(CREATE, context().find(button("CREATE"))),
+                entry(POOL_NAME, context().find(byText("Pool name:"))
+                        .find(byXpath("following-sibling::div//input"))),
+                entry(STARTS_ON, context()
+                        .find(byXpath(".//span[.='Starts on:']/following-sibling::div/div[@role='combobox']"))),
+                entry(STARTS_ON_TIME, context()
+                        .find(byXpath(".//span[.='Starts on:']/following-sibling::span/input"))),
+                entry(ENDS_ON, context()
+                        .find(byXpath(".//span[.='Ends on:']/following-sibling::div/div[@role='combobox']"))),
+                entry(ENDS_ON_TIME, context()
+                        .find(byXpath(".//span[.='Ends on:']/following-sibling::span/input"))),
+                entry(INSTANCE_TYPE, context().find(byXpath(".//div[.='Instance type']"))),
+                entry(CLOUD_REGION, context().find(byXpath(".//div[.='Region']"))),
+                entry(DISK, context().find(byText("Disk:"))
+                        .find(byXpath("following-sibling::div//input"))),
+                entry(AUTOSCALED, context().find(byText("Autoscaled:"))
+                        .parent().find(byClassName("ant-checkbox")))
+        );
+
+        public CreateHotNodePoolAO(HotNodePoolsAO parentAO) {
+            super(parentAO);
+        }
+
+        public CreateHotNodePoolAO setScheduleTime(Primitive timeBox, String time) {
+            click(timeBox);
+            setValue($(byXpath(".//div[@class='ant-time-picker-panel-inner']//input[@placeholder='Select time']")),
+                    time);
+            return this;
+        }
+
+        public CreateHotNodePoolAO setAutoscaledParameter(String param, int size) {
+            setValue(context().find(byText(format("%s:", param)))
+                    .find(byXpath("following-sibling::div//input")), Integer.toString(size));
+            return this;
+        }
+
+        public CreateHotNodePoolAO addDockerImage(String registry, String group, String tool) {
+            context().find(byText("Add docker image")).parent().click();
+            context().find(byXpath(".//div[.='Docker image']")).click();
+            $(byClassName("ant-select-dropdown-menu"))
+                    .shouldBe(enabled)
+                    .findAll(byClassName("ant-select-dropdown-menu-item"))
+                    .stream()
+                    .filter(el -> el.findAll(byXpath(".//div/span")).texts()
+                            .containsAll(Arrays.asList(registry, group, nameWithoutGroup(tool))))
+                    .findFirst()
+                    .get()
```

Comment:
Add a check for Optional before get(), please

Reply:
Done